code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -103,6 +103,53 @@
 	poison_per_bite = 5
 	move_to_delay = 5
 
+//vipers are the rare variant of the hunter, no damage but so much poison medical care will be needed fast. Low hp, high speed and no damage so hit and run tactics.
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper
+	desc = "Furry and black, it makes you shudder to look at it. This one has effervescent purple eyes."
+	icon_state = "hunter"
+	icon_living = "hunter"
+	icon_dead = "hunter_dead"
+	maxHealth = 40
+	health = 40
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	poison_per_bite = 30
+	move_to_delay = 4
+	poison_type = "polonium" //slow acting but this at 30u per bite will seriously mess you up, perfect viper poison

```

Comment:
This gives RADS

Reply:
yeah, rads are slow acting and do horrible effects if you don't get medical attention, not a terrible one for the viper

Reply:
i'm making a new one though